src/GameController/GCExtendedGamepadSnapshot.cs

```diff
@@ -47,6 +47,12 @@ public struct GCExtendedGamepadSnapShotDataV100 {
 		public float /* float_t = float */ LeftTrigger;
 		public float /* float_t = float */ RightTrigger;
 
+		// radar: https://trello.com/c/7FoGTORD (GCExtendedGamepadSnapShotDataV100 struct size / alignment not backward compatible)
```

Comment:
Minor but that can be deleted now.

Reply:
yes, but it would be more useful in master than in a throw-away branch :)